### DISCUSSION TITLE
Let players know why they get damaged when running with shears.

### DISCUSSION THREAD

**Discussion by EuanM199** [PurpurMC/Purpur#461] [__Completed__] [answered]
Currently running with shears just damages the player, this makes the player very confused as they do not understand it, could there be an addition to this to set a message to tell the player why they are being hurt in chat.

maybe something like

"stop running with scissors" in chat.

**Comment by BillyGalbreath** [comment]
What if I put a message in the actionbar instead of chat? It could get pretty spammy in the chat without a cooldown, and if I add a message cooldown for chat it could be forgotten (you know kids and their 5 second attention spans)

**Reply by YouHaveTrouble** [reply]
Completely agree on the action bar, makes more sense and actually wanted to suggest that here but forgot lol

**Reply by ghost** [reply]
Why not instead give the ability to have all messages be sended via the actionbar/title/bossbar/toast etc. Example if someone puts !actionbar in front of the message it would be send on the actionbar.

**Comment by BillyGalbreath** [comment, marked as answer]
Implemented in build 1252 https://github.com/pl3xgaming/Purpur/commit/26556570466890defc2c78330c1cb7a8c1104707
